libs/langchain-community/src/vectorstores/upstash.ts

```diff
@@ -203,7 +272,7 @@ export class UpstashVectorStore extends VectorStore {
   static async fromTexts(
     texts: string[],
     metadatas: UpstashMetadata | UpstashMetadata[],
-    embeddings: EmbeddingsInterface,
+    embeddings: EmbeddingsInterface | "UpstashEmbeddings",
```

Comment:
We've been using `FakeEmbeddings` like this - could we do the same here?

https://github.com/langchain-ai/langchainjs/blob/main/libs/langchain-community/src/vectorstores/vectara.ts#L176

Reply:
Oh, I mean instead of passing a string here, pass `FakeEmbeddings`

Reply:
The thing is, passing something explicit like "UpstashEmbeddings" makes it easier to understand you have the option to either get the embeddings from Upstash, or passing a embedding client to create the embeddings your self. In the Vectara case, it seems that there's no option to pass an embedding, so all the embeddings will be created directly by it self. With Upstash, this is optional. Sorry for the delay here.